### PR TITLE
chore: yarn 3

### PR DESCRIPTION
**Pull request by probablyup**

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -45,7 +45,6 @@
     "rehype-img-size": "^0.0.1",
     "rehype-raw": "^5.0.0",
     "rehype-sanitize": "^4.0.0",
-    "rehype-shiki": "^0.0.9",
```
Comment (probablyup) [inline website/package.json:48]: Not used and was causing issues

File: website/src/index.d.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
+declare module 'gray-matter';
```
Comment (probablyup) [inline website/src/index.d.ts:1]: Missing stub

**Comment by jaredpalmer** [comment]
We should move to pnpm but this is fine for short term

**Comment by probablyup** [comment]
> We should move to pnpm but this is fine for short term

I'd actually say bun if we intended to move off yarn

**Pull request closed without merging**